addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -87,111 +87,6 @@ var MockMailService = Class.extend({
     },
 });
 
-/**
- * Patch all the mailUtils.clearTimeout and mailUtils.setTimeout.
- *
- * @return {Object} helper functions, including unpatch and time management tools.
- */
-var patchMailTimeouts = function () {
-    var currentTime = 0;
-    var timeouts = {};
-    var countTimeout = 0;
-
-    mailUtils.clearTimeout = function (id) {
-        delete timeouts[id];
-    };
-
-    mailUtils.setTimeout = function (func, duration) {
-        duration = duration || 0;
-        var executeTime = currentTime + duration;
-        countTimeout++;
-        timeouts[countTimeout] = {
-            executeTime: executeTime,
-            func: func
-        };
-        return countTimeout;
-    };
```

Comment:
- both `mail.utils` helpers <`set`/`clear`>`Timeout()` no longer used elsewhere, so they should also be removed
- `mail.utils` no longer used in this file, so remove related `require()`